src/Servers/Kestrel/Core/src/Internal/Http/HttpProtocol.Generated.cs

```diff
@@ -61,7 +61,6 @@ private void FastReset()
             _currentIHttpRequestTrailersFeature = this;
             _currentIHttpConnectionFeature = this;
             _currentIHttpMaxRequestBodySizeFeature = this;
-            _currentIHttpMinRequestBodyDataRateFeature = this;
```

Comment:
What happened here?

Reply:
Ah this isn't implemented on HttpProtocol?

Reply:
Isn't implemented by `HttpProtocol` so added it to the `OnReset()` methods of `Http1Connection`, `Http2Stream`, and `Http3Stream` instead which do implement it

Reply:
Got it.

Reply:
Does it make sense to just implement it since it's on all derived types?

Reply:
Could make them abstract implementations on `HttpProtocol`; but don't know how virtual interface implementations behave performance wise

Reply:
OK. Let's leave it.